### DISCUSSION TITLE
`zsh: bus error  cfclient` when Estimate Geometry with Lighthouse

### DISCUSSION THREAD

**Discussion by r-bt** [bitcraze/discussions#1293] [Q&A] [answered]
I have setup two v2 Lighthouse base stations and was going through the Estimate Geometry flow. However upon pressing the **Estimate Geometry** button at the end (shown in fig):
<img width="580" alt="Screenshot 2024-04-18 at 4 57 04 PM" src="https://github.com/bitcraze/discussions/assets/2939329/c9a5e571-c7e3-4240-b212-794e0405f12f">

The cfclient immediately crashed and in the console I get the error:
```
zsh: bus error  cfclient
```

**Comment by gemenerik** [comment]
I managed to run the calibration on M1 with brew installed Python 3.12 with latest client and lib installed through pip.

Can you share some more information like what OS you are running, and Python, cfclient and cflib version?

**Reply by r-bt** [reply]
I am using

- macOS 14.4
- Python 3.11.7 (I tried using 3.12.0 but I couldn't see any USB devices in cfclient)
- Cfclient 2024.3
- Cflib 0.1.25.1

**Reply by knmcguire** [reply]
Hi!

We need some more information aobut:

The [cfclients](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/)'s debug tab (found under 'Help'>'about') has more information your development environment and your connected Crazyflie, so could you copy past that here?

**Reply by dizzy-s** [reply]
I also ran into the exact same problem with Mac OS. Please see below:

Cfclient version: 2024.3
System: darwin
Python: 3.11.9
Qt: 6.7.0
PyQt: 6.7.0

Interface status
radio: Crazyradio version 3.2
UsbCdc: No information available
udp: None
prrt: No information available
cpx: None

Input readers
PySDL2 (0 devices connected)

Input devices
None

Crazyflie
Connected: radio://0/80/2M
Firmware: e527895e8c6e (CLEAN)

Decks found
bcLighthouse4: rev=D, adr=0DFFE7F10000001B

Sensors found
BMP388: 1
imuPhi: 0.0
imuTheta: 0.0
imuPsi: 0.0
AK8963: 0
LPS25H: 0

Sensors tests
MPU6500: 0
AK8963: 0
LPS25H: 0
imuPhi: 0.0
imuTheta: 0.0
imuPsi: 0.0

**Reply by ataffanel** [reply]
@r-bt @dizzy-s Thanks for the system information, I managed to reproduce the problem on an M1 mac with Python 3.11. I will investigate the problem and attempt to solve it.

**Reply by ataffanel** [reply]
I created an [issue in the client repos](https://github.com/bitcraze/crazyflie-clients-python/issues/714), I cannot see a quick easy fix for it so for now using another computer would be the best bet unfortunatly. Maybe installing brew/python for x86 using rosetta might help, but I have not had time to test yet.

**Comment by Fygas1** [comment]
having the same issue on M3

**Reply by knmcguire** [reply]
Could you report this in the ticket opened about this issue? this would be important information: https://github.com/bitcraze/crazyflie-clients-python/issues/714

**Comment by r-bt** [comment, marked as answer]
See https://github.com/bitcraze/crazyflie-clients-python/issues/714#issuecomment-2440151780 for solution
